packages/amazon/src/serverGroup/configure/wizard/instanceType/advancedMode/advancedMode.less

```diff
@@ -0,0 +1,110 @@
+.advanced-mode-selector {
+  .instance-profile-header {
+    display: inline-block;
```

Comment:
Can you use `flex` here instead? That is the preferred display to help us make everything more responsive.

Reply:
Changing styling will change the look-and-feel of `advanced mode` compared to `simple mode`(previous version). Discussed offline and agreed not to use flex.